package.json

```diff
@@ -24,6 +24,10 @@
   "dependencies": {
     "@babel/runtime": "^7.21.5",
     "@codemirror/lang-javascript": "^6.1.5",
+    "@cypress/vite-dev-server": "^5.0.5",
```

Comment:
I believe there are some deps that can be moved to `devDependencies`. This isn't a big deal, but more for `package.json` hygiene

Feel free to do this in a follow up PR